docs/scos/dev/upgradability-services/new-documentation/method-of-extended-class-overridden-on-project-level.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Method of extended class is overridden one the project level
+description: Reference information for evaluator and upgrader tools.
+last_updated: Mar 23, 2022
+template: concept-topic-template
+---
+
+## Method of an extended class is overridden on the project level
+
+Factory, Dependency Provider, Repository, and Entity Manager methods belong to the private API.
+If you extend a core class and override one of its methods, minor releases can cause errors or unexpected changes in functionality.
+
+#### Using custom methods on the project level
+
+To avoid unexpected issues and achieve the same result, instead of overriding the core methods, introduce custom ones.
```

Comment:
add inform block "Meanwhile we are working on introducing a way to report such cases and add more extension points in the core"